deploy_polygon/02_asset/01_deploy_polygon_asset_erc721.ts

```diff
@@ -36,5 +43,5 @@ func.tags = [
   'L2',
   'PolygonAsset',
 ];
-func.dependencies = ['TRUSTED_FORWARDER'];
+func.dependencies = ['TRUSTED_FORWARDER', 'polygonOperatorFilterSubscription'];
```

Comment:
Need to review dependencies in all deploy files, as we have TRUSTED_FORWARDER dependency but refer to TRUSTED_FORWARDER_V2 above

Reply:
TRUSTED_FORWARDER  Tag deploys both TRUSTED_FORWARDER and TRUSTED_FORWARDER_V2 on polygon deploy